src/olympia/editors/tests/test_helpers.py

```diff
@@ -471,13 +388,11 @@ def test_logs(self):
     def test_notify_email(self):
         self.helper.set_data(self.get_data())
         base_fragment = 'reply to this email or join #amo-editors'
-        for template in ['nominated_to_nominated', 'nominated_to_preliminary',
+        for template in ['nominated_to_nominated',
                          'nominated_to_public', 'nominated_to_sandbox',
-                         'pending_to_preliminary', 'pending_to_public',
-                         'pending_to_sandbox', 'preliminary_to_preliminary',
+                         'pending_to_public', 'pending_to_sandbox',
                          'author_super_review', 'unlisted_to_reviewed',
-                         'unlisted_to_reviewed_auto',
-                         'unlisted_to_sandbox']:
+                         'unlisted_to_reviewed_auto', 'unlisted_to_sandbox']:
```

Comment:
This should at least be a tuple. Sad panda to not being able to use parametrize :(